catalog/video/video-security/src/main/java/org/codice/alliance/video/security/claims/videographer/VideographerClaimsHandler.java

```diff
@@ -100,19 +100,21 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
             ClaimsParameters parameters) {
         ProcessedClaimCollection claimsColl = new ProcessedClaimCollection();
         Principal principal = parameters.getPrincipal();
-        for (Claim claim : claims) {
-            URI claimType = claim.getClaimType();
-            List<String> value = claimsMap.get(claimType);
-            if (value != null) {
-                ProcessedClaim c = new ProcessedClaim();
-                c.setClaimType(claimType);
-                c.setPrincipal(principal);
-                value.forEach(c::addValue);
-                claimsColl.add(c);
-            }
-        }
 
         if (principal instanceof VideographerPrincipal) {
+
+            for (Claim claim : claims) {
+                URI claimType = claim.getClaimType();
```

Comment:
Looking at the [`Claim` code](https://github.com/apache/cxf/blob/master/rt/security/src/main/java/org/apache/cxf/rt/security/claims/Claim.java), there's no guarantee that this can't be `null`, so we probably should check for `null` here and have a unit test for it.